backend/src/database/class_enrollments.go

```diff
@@ -230,23 +230,3 @@ func (db *DB) GetProgramClassEnrollmentsAttendance(page, perPage, id int) (int64
 	}
 	return total, content, nil
 }
```

Comment:
@corypride Nitpick: Can you make it `programId` so that its consistent with how `facilityId` is?